app/scripts/services/dimSettingsService.factory.js

```diff
@@ -83,6 +84,15 @@
       // self destruct timer for quality, hope we can remove this some day...
       savedSettings.disableQuality = new Date('9/19/2016') <= new Date();
 
+      // for now just override itemTags. eventually let users create own?
```

Comment:
Just want to say that this looks great and is pretty amazing how well it fits into the UI.  Be careful about the 8k limit per key in the Chrome Storage sync.  You'll fill it up quickly with plain text entry.